feat: v7.22.0

sethkfman opened the pull request:
## **Description**

Release 7.22.0 PR

[Release Tracker Sheet](https://docs.google.com/spreadsheets/d/1tsoodlAlyvEUpkkcNcbZ4PM9HuC9cEM80RZeoVv5OCQ/edit#gid=300906497&range=A2)

sethkfman committed:
bump versions

sethkfman committed:
updated changelog

github-actions[bot] committed:
chore: cherry-pick #9411 (#9461)

This PR cherry-picks #9411

Co-authored-by: Frank von Hoven <141057783+frankvonhoven@users.noreply.github.com>

Cal-L committed:
chore: Chore/cherry pick #9436 (#9449)

This PR cherry-picks #9436

---------

Co-authored-by: Jonathan Ferreira <44679989+Jonathansoufer@users.noreply.github.com>

katkopel commented:
**Testing iOS v.7.22.0 (1321)** 

Onboarding: custom ETH mainnet
✅ Custom mainnet Scenario 1 (Verifying Custom Mainnet Screen Appears):
✅ Custom mainnet Scenario 2 (Entering an Invalid Mainnet RPC URL):
✅ Custom mainnet Scenario 3 (Entering a valid mainnet RPC URL):
✅ Custom mainnet Scenario 4 (Editing a custom mainnet URL on the settings and privacy view):
✅ Custom mainnet Scenario 5 (Deleting a custom mainnet URL on the settings and privacy view):
https://drive.google.com/file/d/1mLsAHvieYRVfKMak8X7znU0nGBV9gnUR/view 


QR Scanner Regression
✅ From the in-app camera I should be able to scan a valid QR code containing an address:
✅ I should be able to use the QR scanner on the send flow to scan an address: 
https://drive.google.com/file/d/11VVW89DBWRKJexv0Q-6zhGVsEfgQXTEB/view

✅ When I scan a dapp QR code using the in-app camera I should see the appropriate warning message: 
✅ When I scan a dapp QR code using the QR scanner on the send flow I should see the appropriate error message: 
https://drive.google.com/file/d/1mLsAHvieYRVfKMak8X7znU0nGBV9gnUR/view?usp=sharing

❌ When I scan an invalid address QR code using the in app camera I should see the appropriate warning message: 
	❗Scanning invalid QR from the in-app camera produces `Invalid deeplink` errors

✅ When I scan an invalid address QR code using the QR scanner on the send flow I should see the appropriate warning message:
https://drive.google.com/file/d/10FF_6BQd5PH4O6wRFVbX-A8v2_uUL1sk/view

❌ A user can scan a private key QR code using the in-app camera to import an account: 
❗Import is successful but throws `Invalid deeplink/URL` error followed by the account’s private key string
https://drive.google.com/file/d/1InA1_wPjODUwxrjqOrSER1l-3gR8e0hM/view 

❌ User scans MMM Receive QR code on different network: 
❗”Network with chain id 0x1 not found your wallet” error when scanning Eth receive QR from another network
https://drive.google.com/file/d/1OjK34MsogEyVUIXOrC_UEkkABGe_pOtC/view 


General view
✅ Scenario: Changing the device language from English to Russian: 
❗Your Wallet is Ready open modal is in English
https://drive.google.com/file/d/1VREQs_EKOzc_LdPswj6_v4kM9rkzsqcT/view

✅ Scenario: Changing the app language from English to Filipino:
https://drive.google.com/file/d/1Xwi2Gxe9eRbi8go7jpCcheKwgs8HlgX_/view


Browser Regression
✅ Scenario: Searching an invalid url and prompts the user with an error message:
✅ Scenario: Navigation with browser controls:
✅ Scenario: ENS website loads correctly:
✅ Scenario: From the browser I can successfully upload a picture:
✅ Scenario: Address bar:
https://drive.google.com/file/d/1LxdVWjDmL2oyX6MvpSIf24uz6JRt_rcV/view

✅ Scenario: Adding browser Favorites:
✅ Scenario: Browser Options Menu:
✅ Scenario: Visiting a malicious website prompts the user with the phishing detection warning:
✅ Scenario: Navigate to DappsMarketplace and connect wallet account:
https://drive.google.com/file/d/1srvanDsg35pgP0uMaNy6bp5XdTX3gwQ8/view


Network Regression
✅ Scenario: I can remove a custom network (Palm) that was added via the popular network flow:
✅ Scenario: I can remove a custom network that was added via the custom network flow.
✅ Scenario Outline: Adding a network via a Dapp:
✅ Scenario Adding a network via the new popular network flow:
✅ Scenario: Adding a network via the custom network flow:
✅ Scenario: Adding a custom network and editing the network:
✅ Scenario: Editing a popular network:
https://drive.google.com/file/d/17XiQbaIzTpcCkkyryXR_q5hUl9sXMwo7/view

Nick-Son commented:
Testing Android v.7.22.0

Onboarding: custom ETH mainnet
✅ Custom mainnet Scenario 1 (Verifying Custom Mainnet Screen Appears):
✅ Custom mainnet Scenario 2 (Entering an Invalid Mainnet RPC URL):
✅ Custom mainnet Scenario 3 (Entering a valid mainnet RPC URL):
✅ Custom mainnet Scenario 4 (Editing a custom mainnet URL on the settings and privacy view):
✅ Custom mainnet Scenario 5 (Deleting a custom mainnet URL on the settings and privacy view):
https://drive.google.com/file/d/1tLWIEg5-bb0CIlOaPBgUSc9djkQKIp7U/view?usp=drive_link


QR Scanner Regression
✅ From the in-app camera I should be able to scan a valid QR code containing an address:
https://drive.google.com/file/d/1sFLQ5wf1Opn3K5wH9MFfgJErhu2pMXh4/view?usp=drive_link

✅ I should be able to use the QR scanner on the send flow to scan an address:
https://drive.google.com/file/d/1sEAl_kfhQ08bGt8_OpGXEma-LJFSgSut/view?usp=drive_link

✅ When I scan a dapp QR code using the in-app camera I should see the appropriate warning message:
https://drive.google.com/file/d/1s8qPIrkPeuvMuNSratEibzcXEZ-7NftO/view?usp=drive_link

✅ When I scan a dapp QR code using the QR scanner on the send flow I should see the appropriate error message:
https://drive.google.com/file/d/1s7M_aP34wFysYPBLSqVFrWVHNeQlqgIm/view?usp=drive_link

✅  When I scan an invalid address QR code using the in app camera I should see the appropriate warning message:
https://drive.google.com/file/d/1s5oi0C4_hJuMuxADrKhuaKIlpkKW11ou/view?usp=drive_link

✅ When I scan an invalid address QR code using the QR scanner on the send flow I should see the appropriate warning message:
https://drive.google.com/file/d/1s2kHbUhwo4fMCLnfq1q1FXrLBox25jHg/view?usp=drive_link

❌ A user can scan a private key QR code using the in-app camera to import an account:
❗Import is successful but throws Invalid deeplink/URL error followed by the account’s private key string
https://drive.google.com/file/d/1s1WqTSVcB4ybqu0oHaHvrc3BuFBduIqa/view?usp=drive_link

❌ User scans MMM Receive QR code on different network:
❗”Network with chain id 137 not found your wallet” error when scanning Eth receive QR from another network
https://drive.google.com/file/d/1rynAy6t3WhWe4JM3QPlucwmG6F_Nmbbe/view?usp=drive_link 


General view
✅ Scenario: Changing the device language from English to Russian:
https://drive.google.com/file/d/1rrSNw5-CiW1oO7L0n5TmCCbA4pfz6Kx_/view?usp=drive_link

✅ Scenario: Changing the app language from English to Filipino:
https://drive.google.com/file/d/1rqk7JKlOO3OhDSe3DqXNIy4IZXnV6OLe/view?usp=drive_link


Browser Regression
✅ Scenario: Searching an invalid url and prompts the user with an error message:
https://drive.google.com/file/d/1sxslh6zOAOrVn1po4NRCgxWGM6MSopMy/view?usp=drive_link

✅ Scenario: Navigation with browser controls:
https://drive.google.com/file/d/1swjlum7TOmq4B96IiEMj4Ew3DDsl6PuS/view?usp=drive_link

✅ Scenario: ENS website loads correctly:
https://drive.google.com/file/d/1suk51TuJaS2JhTzjYQPvMObQMeCz1COg/view?usp=drive_link

✅ Scenario: From the browser I can successfully upload a picture:
https://drive.google.com/file/d/1stDeCLFeicCoYgCGkA4es2aufGsBJGJa/view?usp=drive_link

✅ Scenario: Address bar:
https://drive.google.com/file/d/1sm3O_YEmV0I-AkgRUVrJQmdTXZ6OrV3L/view?usp=drive_link

✅ Scenario: Adding browser Favorites:
https://drive.google.com/file/d/1si_rva05JaRL2OYY9luUGpFvvhX0JDQt/view?usp=drive_link

✅ Scenario: Browser Options Menu:
https://drive.google.com/file/d/1saAtYu6WiGhO9EI92bsdtPc_8mV7OoGY/view?usp=drive_link

✅ Scenario: Visiting a malicious website prompts the user with the phishing detection warning:
https://drive.google.com/file/d/1sVpvsiOlRcT6mK964KJpBo0k9RxSyT7S/view?usp=drive_link

✅ Scenario: Navigate to DappsMarketplace and connect wallet account:
https://drive.google.com/file/d/1sP4sTmE7S6xnmcGydecN82fqLqJTVgbG/view?usp=drive_link


Network Regression
✅ Scenario: I can remove a custom network that was added via the popular network flow:
https://drive.google.com/file/d/1tHZlvT5d8LrrHW--cg2beqCxlmtckY83/view?usp=drive_link

✅ Scenario: I can remove a custom network that was added via the custom network flow:
https://drive.google.com/file/d/1tDWOIYaXkz0moQQJELjqdQCEaPqoAyQm/view?usp=drive_link

✅ Scenario Outline: Adding a network via a Dapp:
https://drive.google.com/file/d/1tCW6emuEYGP6uTjhPAGlUWfEHlsUY7a0/view?usp=drive_link

✅ Scenario Adding a network via the new popular network flow:
https://drive.google.com/file/d/1t7stiqxJETffJus0YmZEAjt3S6S56ThX/view?usp=drive_link

✅ Scenario: Adding a network via the custom network flow:
https://drive.google.com/file/d/1t7WBrRpRFRQZw733aua-JX6r8dJfC_Ic/view?usp=drive_link

✅ Scenario: Adding a custom network and editing the network:
https://drive.google.com/file/d/1t0Cs-VqpAWqmc50oVpoFY9hFDzR8Jlq8/view?usp=drive_link

✅ Scenario: Editing a popular network:
https://drive.google.com/file/d/1sz_QT76W5rrTmqs1Muy_jhOaJmTY0CSl/view?usp=drive_link

plasmacorral commented:
🟢 Approval from Accounts team

github-actions[bot] committed:
chore: cherry-pick #9514 (#9516)

This PR cherry-picks #9514

---------

Co-authored-by: Aslau Mario-Daniel <marioaslau@gmail.com>
Co-authored-by: João Loureiro <1649425+jpcloureiro@users.noreply.github.com>
Co-authored-by: sethkfman <10342624+sethkfman@users.noreply.github.com>

MarioAslau committed:
chore: bump build to 1323 (#9518)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

frankvonhoven committed:
fix: cp url fix (#9538)

## **Description**

Incorporates some UI changes and URL updates for basic functionality

## **Related issues**

Relatd: [to this PR in
main](https://github.com/MetaMask/metamask-mobile/pull/9489)

## **Manual testing steps**

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

salimtb committed:
Cherry pick revert linea goerli network (#9534)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

github-actions[bot] committed:
chore: cherry-pick #9539 (#9556)

This PR cherry-picks #9539

Co-authored-by: Jonathan Ferreira <44679989+Jonathansoufer@users.noreply.github.com>
Co-authored-by: legobeat <109787230+legobeat@users.noreply.github.com>
Co-authored-by: sethkfman <Seth.Kaufman@consensys.net>

sethkfman committed:
chore: RC - 3 bump version 1325 (#9557)

## **Description**

Version bump RC - 3 1325 

## **Related issues**

Fixes: NA

## **Manual testing steps**

1. NA
2.
3.

## **Screenshots/Recordings**

NA

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

MarioAslau committed:
chore: merge main 7.22.0 (#9591)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

---------

Co-authored-by: legobeat <109787230+legobeat@users.noreply.github.com>
Co-authored-by: tommasini <46944231+tommasini@users.noreply.github.com>
Co-authored-by: Curtis David <Curtis.David7@gmail.com>
Co-authored-by: Pedro Pablo Aste Kompen <wachunei@gmail.com>
Co-authored-by: salimtb <salim.toubal@outlook.com>
Co-authored-by: Jonathan Ferreira <44679989+Jonathansoufer@users.noreply.github.com>
Co-authored-by: Jyoti Puri <jyotipuri@gmail.com>
Co-authored-by: Brian August Nguyen <brianacnguyen@gmail.com>
Co-authored-by: Frank von Hoven <141057783+frankvonhoven@users.noreply.github.com>
Co-authored-by: Derek Brans <dbrans@gmail.com>
Co-authored-by: LeoTM <1881059+leotm@users.noreply.github.com>
Co-authored-by: EtherWizard33 <165834542+EtherWizard33@users.noreply.github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: sahar-fehri <sahar.fehri@consensys.net>
Co-authored-by: Desi McAdam <desi.mcadam@gmail.com>
Co-authored-by: Mark Stacey <markjstacey@gmail.com>
Co-authored-by: Cal Leung <cleun007@gmail.com>
Co-authored-by: Nico MASSART <NicolasMassart@users.noreply.github.com>
Co-authored-by: yande <110056475+Andepande@users.noreply.github.com>
Co-authored-by: gantunesr <gustavoantunesrod@gmail.com>
Co-authored-by: Gustavo Antunes <17601467+gantunesr@users.noreply.github.com>
Co-authored-by: gantunesr <gantunesr@users.noreply.github.com>
Co-authored-by: Daniel Cross <dan.s.cross@icloud.com>
Co-authored-by: João Loureiro <1649425+jpcloureiro@users.noreply.github.com>
Co-authored-by: tommasini <tommasini15@gmail.com>
Co-authored-by: OGPoyraz <omergoktugpoyraz@gmail.com>
Co-authored-by: Kylan Hurt <kylanhurt@users.noreply.github.com>
Co-authored-by: sethkfman <10342624+sethkfman@users.noreply.github.com>
Co-authored-by: sethkfman <Seth.Kaufman@consensys.net>
Co-authored-by: Vinicius Stevam <vinicius.stevam@consensys.net>
Co-authored-by: Vinicius Stevam <45455812+vinistevam@users.noreply.github.com>
Co-authored-by: legobt <6wbvkn0j@anonaddy.me>
Co-authored-by: CW <chris.wilcox@consensys.net>
Co-authored-by: Erik Marks <25517051+rekmarks@users.noreply.github.com>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>
Co-authored-by: Charly Chevalier <charlyy.chevalier@gmail.com>
Co-authored-by: Nicolas Ferro <nicolaspatricioferro@gmail.com>
Co-authored-by: Omri Dan <61094771+omridan159@users.noreply.github.com>

sethkfman committed:
resolve conflicts with main

sethkfman committed:
updated from conflict fixes`

sethkfman committed:
updated snapshot for removal of goreil

MarioAslau committed:
chore:  Merge main changes into release/7.22.0 (#9607)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution? -->

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-mobile/issues/9594

## **Manual testing steps**

1. Go to this page... 2. 3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

---------

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution? -->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page... 2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

Co-authored-by: tommasini <tommasini15@gmail.com>
